bin/zopen-generate

```diff
@@ -110,7 +110,7 @@ mkdir -p ${name}port/patches
 
 # Create external LICENSE
 touch "${name}port/patches/LICENSE" && chtag -tc 819 "${name}port/patches/LICENSE"
-echo "${licenseName}\nView license contents at ${licenseUrl}" > "${name}port/patches/LICENSE"
+echo -e "${licenseName}\nView license contents at ${licenseUrl}" > "${name}port/patches/LICENSE"
```

Comment:
The z/OS /bin/echo does not process -e:
```
/bin/echo -e bla
-e bla
```
Can we try printf?

Reply:
A trailing newline is needed..

```shell
 ~  echo "\nView license contents at" > LICENSE && cat LICENSE   
\nView license contents at
 ~  printf "\nView license contents at\n" > LICENSE && cat LICENSE 

View license contents at
 ~  printf "\nView license contents at" > LICENSE && cat LICENSE   

View license contents at ~  

```